### PR TITLE
docker, docker-build, docker-compose: translate to Turkish (tr)

### PR DESCRIPTION
**Pull request by Yutyo**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

PS: It accidentally includes past commits.

### CODE REVIEW

**Comment by bl-ue** [comment]
Hi @Yutyo!  It's great that you've opened 5 PRs already for `tr` translation!  I hate to ask this, but can we close all 5 of the PRs you just opened, and then will you please re-create them, each on its own branch **based off `master`**?  Because it's really difficult to review and especially merge these ones as they are, since they each incrementally add the other's page...

The reason you have all of those previous commits is that wheny ou create the branch for each PR, you were on the branch that you had created for your previous PR, but you should've been on the the `master` branch.

What do you think? It really shouldn't be very much work at all...

**Comment by navarroaxel** [comment]
@bl-ue we can work with all the Turkish translations in one PR. Maybe can just pick the last one and help to @Yutyo to resolve the conflicts.

**Comment by bl-ue** [comment]
Good idea.  So I'll close the other unmerged-ones...

File: pages.tr/common/docker-compose.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# docker-compose
```
Comment (bl-ue) [inline pages.tr/common/docker-compose.md:1]:
```suggestion
# docker compose
```

File: pages.tr/common/docker-compose.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`docker-compose up -d`
+
+- Tüm konteynerleri çalıştırın ve gerekiyorsa yeniden oluşturun:
+
+`docker-compose up --build`
+
+- Tüm konteynerleri alternatif bir beste dosyasıyla başlatın:
+
+`docker-compose --file {{path/to/file}} up`
```
Comment (bl-ue) [inline pages.tr/common/docker-compose.md:20]: Will you please translate `path/to/file`?

**Review by bl-ue — CHANGES_REQUESTED** [review]
Thank you for the page update @Yutyo 👍🏻   I've left some comments below, for you to review.

File: pages.tr/common/docker-compose.md (hunk 2 above)
Comment (Yutyo) [reply pages.tr/common/docker-compose.md:20]: Sure

**Comment by Yutyo** [comment]
I did the requested changes.

**Review by bl-ue — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Pull request merged**